src/modules/sensors/voted_sensors_update.cpp

```diff
@@ -921,6 +919,40 @@ bool VotedSensorsUpdate::check_failover(SensorData &sensor, const char *sensor_n
 
 				// reduce priority of failed sensor to the minimum
 				sensor.priority[failover_index] = 1;
+
+				// Update the subsystem_info uORB given that a sensor failed
+				int ctr_valid = 0;
+
+				for (uint8_t i = 0; i < sensor.subscription_count; i++) {
+					if (sensor.priority[i] > 1) { ctr_valid++; }
+
+					PX4_WARN("FAILOVER event (idx=%u)! Sensor %s: Nr. %u Priority: %u", failover_index, sensor_name, i, sensor.priority[i]);
+				}
+
+				PX4_INFO("%s sensor switch from #%i", sensor_name, failover_index);
```

Comment:
PX4_ERR instead of PX4_INFO?

Reply:
Your call. If we consider switching from one accel to another (which is of course still valid) an error then yes, i can change that. Just let me know.

Reply:
PX4_INFO is a console message that isn't going to be seen. PX4_ERR (and PX4_WARN) are logged. For most of these they either need to be important enough to be logged, or might as well not be included at all.